en/sensor/airspeed.md

```diff
@@ -14,7 +14,7 @@ Recommended digital airspeed sensors include:
   - MEAS Spec series (e.g. [MS4525DO](https://www.te.com/usa-en/product-CAT-BLPS0002.html), [MS5525](https://www.te.com/usa-en/product-CAT-BLPS0003.html))
     - [mRo I2C Airspeed Sensor JST-GH MS4525DO](https://store.mrobotics.io/mRo-I2C-Airspeed-Sensor-JST-GH-p/m10030a.htm) (mRo store)
     - [Digital Differential Airspeed Sensor Kit](https://store-drotek.com/793-digital-differential-airspeed-sensor-kit-.html) (Drotek).
-  - [EagleTree Airspeed MicroSensor V3](http://www.eagletreesystems.com/index.php?route=product/product&product_id=63) (eagletreesystems) <!-- link not working 20230830 -->
+  - EagleTree Airspeed MicroSensor V3 (eagletreesystems) <!-- link not working 20230830 -->
```

Comment:
Yeah, if it has gone that long, link gone. I'll leave you to clean up any special driver stuff they might have.
```suggestion
```